server/plugin/webhook.go

```diff
@@ -937,7 +937,7 @@ func (p *Plugin) postPullRequestReviewCommentEvent(event *github.PullRequestRevi
 
 	post := &model.Post{
 		UserId:  p.BotUserID,
-		Type:    "custom_git_pull_review_comment",
+		Type:    "custom_git_pr_comment",
```

Comment:
While we are not using the custom type currently, I think there is value in keeping the type the same for each post.

What is the reason behind changing it?

Reply:
I kept getting the error that the length of the type should be less than 2, that's why I have reduced it. 
@hanzei